.github/workflows/ci.yml

```diff
@@ -0,0 +1,138 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - "*"
```

Comment:
Is this just because nothing in the build uses the tag (such as if versioning was based on the tag with versioneer, etc) so there's no point in building when a tag is pushed?